### PR TITLE
chore: maintained changelog

### PR DESCRIPTION
**Pull request by Michael-A-Heuer**
## Description

Task: [OS-761](https://aragonassociation.atlassian.net/browse/OS-761)

## Type of change

- [x] documentation

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [x] I have updated the `UPDATE_CHECKLIST` file in the root folder.

[OS-761]: https://aragonassociation.atlassian.net/browse/OS-761?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Michael-A-Heuer** [comment]
To be merged after https://github.com/aragon/osx/pull/477

File: packages/contracts/CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Added missing `virtual` keyword to `PermissionCondition` and `PermissionConditionUpgradeable`.
```
Comment (Rekard0) [inline packages/contracts/CHANGELOG.md:42]: this was not part of 1.4?

File: packages/contracts/CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.4.0-rc0
+## v1.4.0
```
Comment (Rekard0) [inline packages/contracts/CHANGELOG.md:8]: this shouldn't be [upcoming] ?

**Review by mathewmeconry — APPROVED** [review]

File: packages/contracts/CHANGELOG.md [hunk 3]
```diff
@@ -26,21 +26,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed unused `ERC1271Mock` contract.
 - Removed the `setSignatureValidator` function and `signatureValidator` variable in `DAO`. In places, where the function must remain to not alter the `IDAO` interface ID, it will revert and explanatory notes are put in place..
 
-## v1.3.1-rc0
-
-### Added
+## v1.3.1
```
Comment (Michael-A-Heuer) [inline packages/contracts/CHANGELOG.md:29]: We should release this as `aragon/osx@1.3.1` NPM package after we release `aragon/osx@1.3.0`.

File: packages/contracts/CHANGELOG.md (hunk 2 above)
Comment (Michael-A-Heuer) [reply packages/contracts/CHANGELOG.md:8]: Not necessarily. I think it is clearer for us in which release things will end up and if the version number is already written down. However, I am open to change it

**Review by Rekard0 — APPROVED** [review]

**Review by josemarinas — APPROVED** [review]

**Pull request merged**